sdk/communication/Azure.Communication.Rooms/src/RoomsClient.cs

```diff
@@ -325,26 +346,22 @@ public virtual Response<CommunicationRoom> AddParticipants(string roomId, IEnume
         }
 
         /// <summary>
-        /// Add room participants async.
+        /// Add Participants to a Room
```

Comment:
Nit: Seems the descriptions are inverted (Async marked as non-async, non-async marked as async)

Reply:
Fixed. Thanks !